docs/iis.md

```diff
@@ -2,17 +2,51 @@
 
 Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/iis) for instructions to collect metrics from this application using Ops Agent.
 
+Additionally, specify `receiver_version` as `v2` to enable new metrics gathering. 
+
+Example `v2` Configuration
+```yaml
+metrics:
+  receivers:
+    iis:
+      type: iis
+      receiver_version: "2"
+  service:
+    pipelines:
+      iispipeline:
+        receivers:
+          - iis
+```
+
 ## Metrics
 
-The following table provides the list of metrics that the Ops Agent collects from this application.
+The following table provides the list of metrics that the Ops Agent collects from this application for `v1` of this receiver.
```

Comment:
Since this paragraph comes before the version sub-headings:

```suggestion
The following tables provides the lists of metrics that the Ops Agent collects from this application for each supported version of this receiver.
```

Reply:
Updated